### PR TITLE
JNG-4540 Release osgi-utils

### PR DESCRIPTION
**Pull request by gaborflorian**
<table>
<td>
  <a href="https://blackbelt.atlassian.net/browse/JNG-4540" title="JNG-4540" target="_blank">Sub-taskJNG-4540</a>  Release osgi-utils
  </td></table>
  <br />

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -86,7 +86,7 @@
         <karaf-version>4.3.3</karaf-version>
         <karaf-features-version>2.0.1</karaf-features-version>
 
-        <surefire-version>3.0.0-M7</surefire-version>
+        <surefire-version>3.0.0-M5</surefire-version>
```
Comment (bencegelei) [inline pom.xml]: Version downgrade?

**Review by bencegelei — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (gaborflorian) [reply pom.xml]: Something wrong with the surefire plugin

**Review by bencegelei — APPROVED** [review]

**Review by bencegelei — APPROVED** [review]

File: pom.xml (hunk 1 above)
Comment (noherczeg) [inline pom.xml]: Should be M7

**Review by noherczeg — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (gaborflorian) [reply pom.xml]: [1d7f0ba](https://github.com/BlackBeltTechnology/osgi-utils/pull/51/commits/1d7f0ba2ea41d5c3a8b9150dbba80826e7f89443)

**Review by noherczeg — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -482,12 +552,6 @@
                     </configuration>
                 </plugin>
 
-                <plugin>
-                    <groupId>org.sonarsource.scanner.maven</groupId>
-                    <artifactId>sonar-maven-plugin</artifactId>
-                    <version>${sonar-maven-plugin-version}</version>
-                </plugin>
-
```
Comment (robertcsakany) [inline pom.xml:490]: Why sonar is removed? I haven't seen sonar definition at all.

**Review by robertcsakany — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 2 above)
Comment (gaborflorian) [reply pom.xml:490]: duplicated plugin

**Review by Madmat8 — APPROVED** [review]

**Review by robertcsakany — APPROVED** [review]

**Pull request merged**